doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -929,3 +933,99 @@ When an invalid `transient` parameter is given:
 ```
 
 [Top](#top)
+
+##<a name="section7"></a>Metrics
+###<a name="section7.1"></a>`GET /v1/admin/metrics`
+Gets metrics for a whole Cygnus agent. Specifically:
+
+* Number of <b>incoming transactions</b> (a transaction involves a request and a response).
```

Comment:
The precise name of each metric should be included. Maybe the following piece of doc in fiware-orion may help (feel free os copy pasting if you want ;)

https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/metrics_api.md#metrics